tasks/main.yml

```diff
@@ -27,6 +27,14 @@
     group: "{{ minio_group }}"
     shell: /bin/bash
 
+- name: create minio config dir
+  file:
+    path: "{{ minio_config_dir }}"
+    owner: "{{ minio_user }}"
+    group: "{{ minio_group }}"
+    state: directory
+  when: minio_config_dir is defined
```

Comment:
This `when` won't work because it's defined as `""` in the defaults. Let's just set a sane default.